### DISCUSSION TITLE
How can I debug custom commands?

### DISCUSSION THREAD

**Discussion by doureyd** [notaapp/talk#151] [Questions] [answered]
Hello there, how can I debug/see the logs of the custom commands I'm building?

Thanks

**Comment by astoilkov** [comment]
You can run the custom command via the Terminal by passing the argument manually. In the future, we will make it possible to see the output of the command inside Nota.

**Comment by astoilkov** [comment, marked as answer]
We now have a `@debug` syntax that you can use to debug a command — https://docs.nota.md/@debug.html.
